automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.service.Orchestration
+import org.broadinstitute.dsde.workbench.service.test.RandomUtil
+import org.scalatest.{BeforeAndAfterAll, Outcome, fixture}
+
+import scala.util.{Failure, Success, Try}
+
+
+/**
+  * trait BeforeAndAfterAll - One cluster, billing project per Scalatest Spec.
+  */
+abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAll with LeonardoTestUtils with BillingFixtures with RandomUtil with LazyLogging {
```

Comment:
One place that claim clean billing project and create new cluster with `beforeAll` and `afterAll`

Reply:
👍 thanks